### PR TITLE
DCS-1114 Fixing duplicate check and adding more tests

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/data/draftReportClient.test.ts [hunk 1]
```diff
@@ -135,10 +135,11 @@ test('getDuplicateReports', () => {
               ,   r.reporter_name reporter
               ,   r.status status
               from v_report r
-              where r.booking_id >= $1
+              where r.booking_id = $1
               and r.incident_date >= $2
               and r.incident_date <= $3
-              and r.status != $4`,
+              and r.status != $4
+              order by date asc`,
```
Comment (andrewrlee) [inline server/data/draftReportClient.test.ts:142]: fixing query and adding sorting

File: server/data/reportingClient.ts [hunk 2]
```diff
@@ -54,7 +55,7 @@ export default class ReportingClient {
   public async getIncidentsOverview(
     agencyId: AgencyId,
     [startDate, endDate]: DateRange,
-    statuses
+    statuses: LabelledValue[]
```
Comment (andrewrlee) [inline server/data/reportingClient.ts:58]: Adding some type info to fix broken linter

File: server/data/draftReportClient.ts [hunk 3]
```diff
@@ -98,10 +98,11 @@ export default class DraftReportClient {
...
               ,   r.status status
               from v_report r
-              where r.booking_id >= $1
+              where r.booking_id = $1
               and r.incident_date >= $2
               and r.incident_date <= $3
-              and r.status != $4`,
+              and r.status != $4
+              order by date asc`,
       values: [bookingId, startDate, endDate, ReportStatus.IN_PROGRESS.value],
```
Comment (bell-pa) [inline server/data/draftReportClient.ts:106]: Hmm. Perhaps the node equivalent of Testcontainers might improve database client testing.  Does it exist?

**Review by bell-pa — APPROVED** [review]
LGTM

File: server/data/draftReportClient.ts (hunk 3 above)
Comment (andrewrlee) [reply server/data/draftReportClient.ts:106]: as discussed, this would help but quite a big change

**Pull request merged**